### DISCUSSION TITLE
Testing Flexget commands in Docker container on Synology DSM?

### DISCUSSION THREAD

**Discussion by ultimateguy** [Flexget/Flexget#3252] [Q&A]
I'm running the wiserain/flexget Docker container on my Synology NAS. Is there a way to run flexget commands from within this container? I have tried to input commands from the container's terminal in the Docker GUI within DSM, but nothing ever happens when I type commands in this terminal.

**Comment by whiteout12** [comment]
If you ssh into your NAS from a terminal (not DSM gui) you should be able to start a session in your container with `docker exec -it flexget sh`.
When inside you can execute your flexget commands from there
